pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/CompatibilityOpsRunner.java

```diff
@@ -36,8 +35,7 @@ private CompatibilityOpsRunner(String configFileName) {
     _configFileName = configFileName;
   }
 
-  private boolean runOps()
-      throws IOException, JsonParseException, JsonMappingException {
+  private boolean runOps() throws IOException, JsonParseException, JsonMappingException, InterruptedException {
```

Comment:
I'd suggest throwing a generic `Exception` instead of putting all the concrete ones here.

Reply:
done